packages/overlays/test/GlobalOverlayController.test.js

```diff
@@ -154,6 +154,50 @@ describe('GlobalOverlayController', () => {
       await ctrl.sync({ isShown: false, data: { text: 'goodbye world' } });
       expect(getRenderedContainers().length).to.equal(0);
     });
+
+    describe('contentNode instead of a lit template', () => {
+      it('accepts HTML Element (contentNode) as content', async () => {
+        const contentNode = await fixture(
+          html`
+            <p>my content</p>
+          `,
+        );
+
+        const ctrl = overlays.add(
+          new GlobalOverlayController({
+            contentNode,
```

Comment:
what's the use-case? we should never do this as it would require to move user-provided light dom to move to the body... (which will mix up a lot of stuff... like the parents' lit-html template)?

Reply:
I agree it's not ideal to do this, but I was under the impression we should allow this as it is public API. And we kind of need it to combine rich select with global overlays, where the rich select needs to have the DOM (template isn't enough) to function properly. So therefore we figured let's make it work with nodes first and see if we can make it work for templates later.